invenio_rdm_migrator/load/ids.py

```diff
@@ -15,19 +15,21 @@ def generate_uuid(data):
     return str(uuid4())
 
 
-def pid_pk():
+def pid_pk(initial_pid=1_000_000):
     """Generate an autoincrementing numeric primary key."""
     if not hasattr(pid_pk, "value"):
-        pid_pk.value = 1_000_000
+        pid_pk.value = initial_pid
```

Comment:
this is still not working as it gets even in the second run the `1_000_000` id...